src/components/FaceVideo/Challenge.tsx

```diff
@@ -50,22 +52,24 @@ const Challenge: FunctionComponent<Props> = ({ challenge, translate }) => {
 
     return (
       <ChallengeContainer
-        title={translate('video_capture.header.challenge_turn_template', {
-          side: translate(`video_capture.header.challenge_turn_${side}`),
-        })}
-        renderInstructions={() => (
+        title={translate(`video_capture.header.challenge_turn_${side}`)}
+        renderChallenge={() => (
           <span
+            aria-hidden={true}
```

Comment:
Could this be just?
```suggestion
            aria-hidden
```

Reply:
Not very clear from the React documentation if it can be defined that way, but it also looks like the value is meant to be a string. Will make that change.